### DISCUSSION TITLE
Feature Request: Variable Annotations

### DISCUSSION THREAD

**Discussion by colvin** [envkey/envkey#102] [Ideas and feedback]
I would love to have the ability in the EnvKey UI to attach an annotation or note to individual variables, in order to document what the variables are for and/or why the variables have the values that they do. This would allow me to bring documentation for the variables closer to where they are defined and managed.

**Comment by danenania** [comment]
Sorry for the slow response on this @colvin -- I agree this would be a useful feature. Moving this over to discussions.
